src/gov-action-contracts/address-registries/L2GovAddressRegistryInterfaces.sol

```diff
@@ -64,7 +64,7 @@ interface IL2ArbitrumTokenGetter {
     function l2ArbitrumToken() external view returns (IL2ArbitrumToken);
 }
 
-interface IArbOneGovAddressRegistry is
```

Comment:
What about Nova? We may need an address registry there at some point, one reason might be to handle DAC reward changes. What will call that address reg?

Reply:
I think keeping this one for gov-specific addresses and introducing a new registry for anytrust-specific things makes most sense

Reply:
...then again, maybe just L1 vs. L2 address registry is nicest?

Reply:
you convinced me